lib/ssl/test/ssl_pem_cache_SUITE.erl

```diff
@@ -390,12 +390,12 @@ alternative_path_hardlink(Config) when is_list(Config) ->
 alternative_path_symlink() ->
     [{doc,"Test that internal reference table contains expected data for"
       " absolute and symbolic link. "
-      "This test verifies handling of same file with an alternative reference."}].
-%% see alternative_path_hardlink for specification
+      "This test verifies handling of same file with an alternative reference."
```

Comment:
Maybe add some explanation of why counters change in the way they do!? See for instance the manual clean-up case.

Reply:
I tried to improve the comment.